service/discovery/azure/compute.go

```diff
@@ -74,9 +75,52 @@ func (d *azureComputeDiscovery) List() (list []voc.IsResource, err error) {
 	}
 	list = append(list, virtualMachines...)
 
+	// Discover functions
+	function, err := d.discoverFunction()
+	if err != nil {
+		return nil, fmt.Errorf("could not discover functions: %w", err)
+	}
+	list = append(list, function...)
+
 	return
 }
 
+// Discover function
+func (d *azureComputeDiscovery) discoverFunction() ([]voc.IsResource, error) {
+	var list []voc.IsResource
+
+	client := web.NewAppsClient(to.String(d.sub.SubscriptionID))
```

Comment:
Are you sure, this is the correct call for Azure functions? Isn't this referring to https://azure.microsoft.com/en-us/services/app-service/web/?

Reply:
honestly I do not know :) The call above gets the Function App shown here: https://portal.azure.com/#blade/HubsExtension/BrowseResource/resourceType/Microsoft.Web%2Fsites/kind/functionapp

I don´t know if and what is the difference between Azure Functions and Function Apps.

Reply:
You´re right, functions are created in function apps. Ok, than I have to extend that.

Reply:
It looks like it is the correct call, but it seems they are grouped in "App Service Plans", they seem to be be a container for those apps, but I think we can ignore that for now, so it seems that everything looks good for now.